testsuite/features/support/env.rb

```diff
@@ -254,6 +254,16 @@ def enable_assertions
   skip_this_scenario unless $server_http_proxy
 end
 
+# do test only if the registry is available
+Before('@noauth_registry') do
```

Comment:
Can I ask to use `no_auth_registry` instead?
From my point of view, either we separate all the words with a `_`  either we don't separate them.

Reply:
done